### PR TITLE
Updated torch network with batch_size, still overfitting though

### PR DESCRIPTION
**Pull request by chantsin**
Training accuracy ~ 99%
Validation accuracy ~ 93% 

How to fix this...

### CODE REVIEW

**Comment by clem21chan** [comment]
Great Scores!

**Review by clem21chan — COMMENTED** [review]
Scores are looking great!
